### PR TITLE
Add a sample for using azureFile for PVs

### PR DESCRIPTION
**Pull request by DonMartin76**
The example for azureFile previously just had a sample of how to directly mount an Azure Files storage into a Pod. I thought it would be nice to also have an example which uses a persistent volume/persistent volume claim, which can be done in the same way. I also added how to get the storage account key via code (using the az command line), and how to create a secret using `kubectl` directly instead of via a `Secret` manifest (which is somewhat clunky).

### CODE REVIEW

**Comment by idvoretskyi** [comment]
cc @kubernetes/sig-azure-misc

**Comment by ahmetb** [comment]
Can someone from Azure review?

**Comment by jdumars** [comment]
@khenidak PTAL

File: staging/volumes/azure_file/README.md [hunk 1]
```diff
@@ -6,15 +6,32 @@ Install *cifs-utils* on the Kubernetes host. For example, on Fedora based Linux
 
 Note, as explained in [Azure File Storage for Linux](https://azure.microsoft.com/en-us/documentation/articles/storage-how-to-use-files-linux/), the Linux hosts and the file share must be in the same Azure region.
 
-Obtain an Microsoft Azure storage account and create a [secret](secret/azure-secret.yaml) that contains the base64 encoded Azure Storage account name and key. In the secret file, base64-encode Azure Storage account name and pair it with name *azurestorageaccountname*, and base64-encode Azure Storage access key and pair it with name *azurestorageaccountkey*.
+## Create a storage access secret
+
+Obtain an Microsoft Azure storage account and create a [secret](secret/azure-secret.yaml) that contains the base64 encoded Azure Storage account name and key. In the secret file, base64-encode Azure Storage account name and pair it with name `azurestorageaccountname`, and base64-encode Azure Storage access key and pair it with name `azurestorageaccountkey`.
+
+Alternatively, use `kubectl` directly to create the secret:
```
Comment (andyzhangx) [inline staging/volumes/azure_file/README.md]: I would suggest use `kubectl create secret` command as first choice

Comment (DonMartin76) [reply staging/volumes/azure_file/README.md]: I agree. I just didn't want to change what was there before, just add things. But I will rewrite it slightly.

Comment (andyzhangx) [reply staging/volumes/azure_file/README.md]: maybe you could change the suggestion sequence :)

Comment (DonMartin76) [reply staging/volumes/azure_file/README.md]: Did so ;-)

**Pull request merged**